src/simulations/optimisation.py

```diff
@@ -50,6 +50,8 @@ def res_post_processing(self, res):
             )
             res['hourly_line_losses'] = res['hourly_line_losses_pu'] * self.grd['base_power'] / 1000
         else:
+            res["voltage_squared"] = np.zeros([1, self.N])
+            res['hourly_line_losses'] = np.zeros(self.N)
```

Comment:
yes thanks for this. Sorry to have caused this issue. I think we duplicate the work as I had fixed this in the main branch.

Reply:
could you please merge the main branch with this branch to harmonise before merging else there will be conflicts as we have solved the same problem and edited the same lines.